handler/src/main/java/io/netty/handler/ssl/ReleasableOpenSslEngine.java

```diff
@@ -0,0 +1,1988 @@
+    private static final Method GET_SERVER_NAMES_METHOD;
+    private static final Method SET_SERVER_NAMES_METHOD;
+    private static final Method GET_ASCII_NAME_METHOD;
+    private static final Method GET_USE_CIPHER_SUITES_ORDER_METHOD;
+    private static final Method SET_USE_CIPHER_SUITES_ORDER_METHOD;
+    private static final AtomicIntegerFieldUpdater<ReleasableOpenSslEngine> refCntUpdater;
+    private static final ResourceLeakDetector<ReleasableOpenSslEngine> leakDetector =
+            ResourceLeakDetectorFactory.instance().newResourceLeakDetector(ReleasableOpenSslEngine.class);
+
+    static {
+        AtomicIntegerFieldUpdater<ReleasableOpenSslEngine> destroyedUpdater =
+                PlatformDependent.newAtomicIntegerFieldUpdater(ReleasableOpenSslEngine.class, "destroyed");
+        if (destroyedUpdater == null) {
+            destroyedUpdater = AtomicIntegerFieldUpdater.newUpdater(ReleasableOpenSslEngine.class, "destroyed");
+        }
+        DESTROYED_UPDATER = destroyedUpdater;
+
+        AtomicIntegerFieldUpdater<ReleasableOpenSslEngine> updater =
+                PlatformDependent.newAtomicIntegerFieldUpdater(ReleasableOpenSslEngine.class, "refCnt");
+        if (updater == null) {
+            updater = AtomicIntegerFieldUpdater.newUpdater(ReleasableOpenSslEngine.class, "refCnt");
+        }
+        refCntUpdater = updater;
+
+        Method getUseCipherSuitesOrderMethod = null;
+        Method setUseCipherSuitesOrderMethod = null;
+        Class<?> sniHostNameClass = null;
+        Method getAsciiNameMethod = null;
+        Method getServerNamesMethod = null;
+        Method setServerNamesMethod = null;
+        if (PlatformDependent.javaVersion() >= 8) {
+            try {
+                getUseCipherSuitesOrderMethod = SSLParameters.class.getDeclaredMethod("getUseCipherSuitesOrder");
+                SSLParameters parameters = new SSLParameters();
+                @SuppressWarnings("unused")
+                Boolean order = (Boolean) getUseCipherSuitesOrderMethod.invoke(parameters);
+                setUseCipherSuitesOrderMethod = SSLParameters.class.getDeclaredMethod("setUseCipherSuitesOrder",
+                        boolean.class);
+                setUseCipherSuitesOrderMethod.invoke(parameters, true);
+            } catch (Throwable ignore) {
+                getUseCipherSuitesOrderMethod = null;
+                setUseCipherSuitesOrderMethod = null;
+            }
+            try {
+                sniHostNameClass = Class.forName("javax.net.ssl.SNIHostName", false,
+                        PlatformDependent.getClassLoader(ReleasableOpenSslEngine.class));
+                Object sniHostName = sniHostNameClass.getConstructor(String.class).newInstance("netty.io");
+                getAsciiNameMethod = sniHostNameClass.getDeclaredMethod("getAsciiName");
+                @SuppressWarnings("unused")
+                String name = (String) getAsciiNameMethod.invoke(sniHostName);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "name" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1481)